doc/transform_methods/rbasex.rst

```diff
@@ -0,0 +1,211 @@
+.. _rBasex:
+
+rBasex
+======
+
+
+Introduction
+------------
+
+This method resembles the pBasex [1]_ approach of expanding a velocity-map
+image over a 2D basis set in polar coordinates, but uses more convenient basis
+functions with analytical Abel transforms, developed by M. Ryazanov in [2]_.
+
+
+How it works
+------------
+
+In velocity-map imaging with cylindrically symmetric photodissociation (in a
+broad sense, including photoionization and photodetachment) the 3D velocity
+distribution at each speed (3D radius) consists of a finite number of spherical
+harmonics :math:`Y_{nm}(\theta, \varphi)` with :math:`m = 0`, which are also
+representable as Legendre polynomials :math:`P_n(\cos\theta)`. This means that
+an :math:`N \times N` image has only :math:`N_r \times N_a` degrees of freedom,
+where :math:`N_r` is the number of radial samples, usually :math:`N / 2`, and
+:math:`N_a` is the number of angular terms, a small number depending on the
+number of photons. These are the “radial distribution” extracted from the
+transformed image in other, general Abel-inversion methods.
+
+However, if these radial distributions are considered as a basis, the 3D
+distribution can be represented as a linear combination of these basis
+functions with some coefficients. And the corresponding image, being the
+forward Abel transform of the 3D distribution, will be represented as a linear
+combination of basis-function projections, that is, their forward Abel
+transforms, with the same coefficients. The reverse is also true: finding the
+expansion coefficients of an experimental VM image over the projected basis
+directly gives the expansion coefficients of the initial 3D velocity direction
+and thus the sought radial distributions.
+
+Finding the expansion coefficients is a simple linear problem, and the forward
+Abel transforms of the basis functions can be calculated easily if the basis is
+chosen wisely.
+
+See :ref:`rBasexmath` for the complete description.
```

Comment:
Well, if you click on [+] to expand “How it works” in the TOC, it will show up there. :–) I'll try to check whether it can be expanded there by default. About including everything in the main document, I don thick that it is a good idea to just put everything in “How it works”. Maybe insert it before “Citation"? But is is really long and heavy.